core/src/js/components/battlegrounds/desktop/categories/hero-details/bgs-last-warbands.component.ts

```diff
@@ -22,7 +22,7 @@ import { normalizeCardId } from '../../../post-match/card-utils';
 	template: `
 		<div class="bgs-last-warbands">
 			<with-loading [isLoading]="false" [mainTitle]="null" [subtitle]="null" svgName="loading-spiral">
-				<ng-container *ngIf="boards$ | async as boards; else emptyState">
+				<ng-container *ngIf="(boards$ | async)?.length && (boards$ | async) as boards; else emptyState">
```

Comment:
Here you usually want to avoid listening to the same observable twice (it will recompute the value for each call).

There is no nice built-in mechanism in Angular for that, so you have to cheat a bit:

```
<div class="bgs-last-warbands" *ngIf="{ boards: boards$ | async } as value">
			<with-loading [isLoading]="false" [mainTitle]="null" [subtitle]="null" svgName="loading-spiral">
				<ng-container *ngIf="value.boards?.length; else emptyState">
					<div
						class="title"
						[owTranslate]="'app.battlegrounds.personal-stats.hero-details.last-warbands.title'"
						[translateParams]="{ value: value.boards?.length }"
					></div>
					<div class="boards" scrollable>
						<div class="board-container" *ngFor="let board of value.boards">
```

I will do the change on my side, so I'll close this PR :)